src/components/chat/ui/ChatInput.tsx

```diff
@@ -82,9 +101,6 @@ const container = css`
         padding-bottom: 8px;
         padding-inline-start: 16px;
         border-radius: 20px;
-        // Not to zoom in on mobile set font-size to 16px which blocks the zooming on iOS
-        // @link: https://weblog.west-wind.com/posts/2023/Apr/17/Preventing-iOS-Safari-Textbox-Zooming
-        font-size: ${isIOSMobile ? 16 : 14}px;
```

Comment:
We cannot use dynamic variables at compile time, so we've moved them to separate styles and are using them at runtime.